spec_mode/mirimrs/notebook1/MRS_Notebook1.ipynb

```diff
@@ -140,14 +140,6 @@
     "- mirisim does not add all of the necessary header keywords for the pipeline to know how to do background subtraction, identify source type, etc.  In order to get these APT-derived keywords correct they will need to be set manually."
```

Comment:
<p>I changed <code>redolong</code> to <code>True</code> because I'm assuming we won't merge a finalized version of this notebook until after the JWebbinars have concluded.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
redolong should be kept as False by default; without it some notebooks take 20+ hours to run, which may be concerning for first-time users who are puzzled why nothing appears to be happening.